File: src/main/kotlin/by/itsupportme/socialnetwork/repositories/UserRepo.java
package by.itsupportme.socialnetwork.repositories;

import by.itsupportme.socialnetwork.beans.User;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepo extends CrudRepository<User, Long> {
    User getAllById(Long id);
}

File: src/main/kotlin/by/itsupportme/socialnetwork/repositories/UserRepo.kt
package by.itsupportme.socialnetwork.repositories

import by.itsupportme.socialnetwork.beans.User
import by.itsupportme.socialnetwork.beans.jwt.JwtRequest
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository


@Repository
interface UserRepo : CrudRepository<User?, Long?> {
    fun findByName(name: String?): User?
}
